browser/brave_vpn/win/brave_vpn_wireguard_service/status_tray/status_icon/constants.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_VPN_WIN_BRAVE_VPN_WIREGUARD_SERVICE_STATUS_TRAY_STATUS_ICON_CONSTANTS_H_
+#define BRAVE_BROWSER_BRAVE_VPN_WIN_BRAVE_VPN_WIREGUARD_SERVICE_STATUS_TRAY_STATUS_ICON_CONSTANTS_H_
+
+#include "base/files/file_path.h"
+
+namespace brave_vpn {
+
+// Status tray icon window name and class.
+constexpr base::FilePath::CharType kStatusTrayWindowName[] =
+    FILE_PATH_LITERAL("BraveVpn_StatusTrayWindow");
+constexpr base::FilePath::CharType kStatusTrayWindowClass[] =
+    FILE_PATH_LITERAL("BraveVpn_StatusTraydowClass");
+const base::FilePath::CharType kBraveVpnStatusTrayMessageName[] =
+    FILE_PATH_LITERAL("CustomTrayMessage");
```

Comment:
Q. I was looking for where this comes from or where it's sent from, but couldn't. Is this Win32 internal message?

Reply:
This is my custom message, system allows to create own messages using https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerwindowmessagea

Reply:
Yeah, my win32 is rusty so I'm not sure but doesn't it make sense when you send message with this custom message like `::SendMessage(hWnd, kBraveVpnStatusTrayMessageName...)` ? I found `SendMessage(hWnd, WM_MENUCOMMAND)` but not sure where this custom message is sent from.

Reply:
and if it's custom message how about adding prefix for consistency with others? `"BraveVpn_CustomTray..."`

Reply:
yeah, nice catch! Renamed and fixed message id